amazon-cloudwatch-container-insights/cloudwatch-agent-dockerfile/source/Dockerfile

```diff
@@ -9,6 +9,8 @@ ARG TARGETARCH
 RUN mkdir -p /go/src/github.com/aws/amazon-cloudwatch-agent/
 WORKDIR /go/src/github.com/aws/amazon-cloudwatch-agent/
 
+ENV GOPROXY=direct
```

Comment:
Can you confirm why this is needed?

Reply:
Forgot to delete it again when building docker image though. Deleted it in the newest commit.